app/controllers/posts_controller.rb

```diff
@@ -26,10 +26,10 @@ def new
 
   def update
     @post = Post.find(params[:id])
-    if @post.update_attributes(post_params)
+    if @post.update(post_params)
```

Comment:
I don't think that's true because this is calling `User#update`, the instance method, not `User.update`, the class method.

According to the docs, [#update will return false](http://api.rubyonrails.org/classes/ActiveRecord/Persistence.html#method-i-update) if the record is not saved.